ivy/__init__.py

```diff
@@ -4,6 +4,15 @@
 from ivy._version import __version__ as __version__
 import builtins
 import numpy as np
+import importlib
+
+
+modules = ["jax", "tensorflow", "torch"]
```

Comment:
The logic looks great, it would be better if we get the modules programmatically so that even if there's a new backend added, it would be taken into account (e.g. paddle and mindspore are being added and we'd like modules to contain those without having to explicitly add it to the list). Secondly, we should do this check at the end of the `__init__.py` file so that even if any other file imported in this file does a framework import, we'd be able to check for it